spec/silencer/rails/logger_spec.rb

```diff
@@ -78,4 +79,14 @@
 
     logger.call(Rack::MockRequest.env_for('/'))
   end
+
+  context 'when logger is Syslogger' do
+    it 'does not throw error' do
+      mock_rails_logger = Syslogger.new
+      allow(::Rails).to receive(:logger) { mock_rails_logger }
+
+      logger = Silencer::Rails::Logger.new(app, silence: ['/'])
+      expect { logger.call(Rack::MockRequest.env_for('/')) }.to_not raise_error
```

Comment:
Would you mind changing this to `not_to`. There's nothing wrong with `to_not` just a personal preference 😄